john-deere-api.py

```diff
@@ -16,6 +16,7 @@
     'clientSecret': '',
     'wellKnown': 'https://signin.johndeere.com/oauth2/aus78tnlaysMraFhC1t7/.well-known/oauth-authorization-server',
     'callbackUrl': 'http://localhost:9090/callback',
```

Comment:
Do we want to extract the base URL as a variable? Not a huge deal, just wondering since we're using it in multiple places now.

Reply:
Just in case someone wants to change it, (ex. different port)

Reply:
I like that idea. Then it's more obvious that the okta callback and connections redirect should be the same server.